resource-service/src/main/java/com/epam/aidial/core/resourceservice/service/ResourceTopic.java

```diff
@@ -39,7 +38,7 @@ public Subscription subscribe(Collection<ResourceDescriptor> resources, Consumer
             String url = resource.getUrl();
             urlToSubscriptions.compute(url, (key, subs) -> {
                 if (subs == null) {
-                    subs = new ConcurrentHashSet<>();
+                    subs = ConcurrentHashMap.newKeySet();
```

Comment:
why?

Reply:
we want to remove vertx dip from the module

Reply:
It comes from java.util.*

Reply:
Sorry, it comes from Vertx